gradle/java.gradle

```diff
@@ -94,12 +94,13 @@ plugins.withId('java') {
         testLogging.showCauses = true
         testLogging.showExceptions = true
         testLogging.showStackTraces = true
-        testLogging.events = ["FAILED"]
-        maxParallelForks = System.properties['maxParallelForks'] ? System.properties['maxParallelForks'].toInteger() : 1
+        testLogging.events = ["STARTED", "PASSED", "SKIPPED", "FAILED"]
```

Comment:
the reason why only FAILED event was present ref: https://github.com/pravega/pravega/pull/5441#pullrequestreview-563051076

Reply:
I removed the duplicate on start.
Given that there is parallelization I don't consider 'passed' and 'started' redundant anymore, because it allows you to infer which are running at the same time.